sdk/storage/azure_storage_blob/src/models/blob_properties.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core::headers::Headers;
+
+/// Properties of an Azure Storage blob.
+///
+#[derive(Clone, Default, Debug)]
+pub struct BlobProperties {
+    pub access_tier_inferred: String,
+    pub access_tier: String,
+    pub blob_type: String,
```

Comment:
Are all these truly just free-form strings or extensible enums, which we cover in the guidelines: https://azure.github.io/azure-sdk/rust_introduction.html#rust-enums